Time/age-clock.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
```

Comment:
This change allows python to be picked up from a variety of places -- macOS install, Homebrew on Intel silicon install, Homebrew on M1 silicon install, etc.

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
```